### PR TITLE
Website updates for new releases

### PR DESCRIPTION
**Pull request by mattcaswell**

### CODE REVIEW

**Review by levitte — APPROVED** [review]

File: news/secadv/20190910.txt [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+OpenSSL 1.0.2 is currently only receiving security updates. Support for 1.0.2
+will end on 31st December 2019.
+
+Support for 1.1.0 ends on 11th September 2019 so 1.1.0l is expected to be the
+last 1.1.0 release.
+
+Users of these versions should upgrade to OpenSSL 1.1.1.
+
+
+Referenses
```
Comment (romen) [inline news/secadv/20190910.txt:97]: Typo

**Review by romen — CHANGES_REQUESTED** [review]

File: news/secadv/20190910.txt (hunk 1 above)
Comment (mattcaswell) [reply news/secadv/20190910.txt:97]: I always put at least one typo in just to check whether people have read it.

Comment (romen) [reply news/secadv/20190910.txt:97]: Did I win something???

Comment (mattcaswell) [reply news/secadv/20190910.txt:97]: > Did I win something???

My congratulations.

**Comment by mattcaswell** [comment]
Pushed.

**Pull request closed without merging**